packages/storybook/stories/index.tsx

```diff
@@ -2,10 +2,9 @@ import "@buildo/bento-design-system/lib/index.css";
 import "@buildo/bento-design-system/lib/defaultTheme.css";
```

Comment:
The changes in this file should be representative of the breaking changes I expect to see in the projects when upgrading